.github/workflows/unit.yml

```diff
@@ -0,0 +1,42 @@
+name: 'Unit'
+
+on:
+  push:
+    branches:
+    - 'main'
+  pull_request:
+    branches:
+    - 'main'
+  workflow_dispatch:
+
+concurrency:
+  group: '${{ github.workflow }}-${{ github.head_ref || github.ref }}'
+  cancel-in-progress: true
+
+jobs:
+  unit:
+    strategy:
+      fail-fast: false
+      matrix:
+        os:
+        - 'ubuntu-latest'
```

Comment:
Per Seths comment, remove the matrix or add the OS matrix here. I would use the same 3 from cloud run

```yaml
os:
- 'ubuntu-latest'
- 'windows-latest'
- 'macos-latest'
```

Reply:
I "temporarily" stripped this back because the 10x MacOS build minutes multiplier was chewing through my allowance... Will re-add 👍